packages/plugin/src/graphql-ast.ts

```diff
@@ -12,10 +12,9 @@ export function getReachableTypes(schema: GraphQLSchema): ReachableTypes {
   if (process.env.NODE_ENV !== 'test' && reachableTypesCache) {
     return reachableTypesCache;
   }
-  // 👀 `printSchemaWithDirectives` keep all custom directives and `printSchema` from `graphql` not
-  const printedSchema = printSchemaWithDirectives(schema); // Returns a string representation of the schema
-  const astNode = parse(printedSchema); // Transforms the string into ASTNode
-  const cache = Object.create(null);
+
+  const astNode = getDocumentNodeFromSchema(schema); // Transforms the schema into ASTNode
```

Comment:
`printSchemaWithDirectives` also uses this function and no need to do `print` and `parse` again.